4 - AnimalShelter/Code/Animal.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AnimalShelter.Code.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AnimalShelter.Code
+{
+    public class Animal
+    {
+        private string _Type = "";
+        
+        public string Type { get => _Type;
+            set
+            {
+                _Type = value.ToLower().Trim();
+                //using a IF for now as the only requirement is to fliter by flight.
+                //Needs refactor if it expands
+                if (_Type == nameof(KnownAnimals.bird))
+                    Feature = nameof(AnimalFeatures.flight);
+            }
+            }
+        public string Feature { get; set; } = "";
```

Comment:
Though I am glad to see an object representing the properties of an Animal, the implementation relies upon strings which is not strongly typed. It is to "loose" and can easily result in bugs down the line. The "Type" of the animal should be an enum or should be a strongly typed class. Same with the feature.